roommanager.js

```diff
@@ -19,17 +19,22 @@ module.exports = function (server, storage) {
 			users: [],
 		};
 
-
 		for (let i = 0; i < room.clients.length; i++) {
+			let ws = room.clients[i].socket;
+
+			// make sure the socket is still open
+			if (ws.readyState != 1) {
+				continue;
+			}
+
 			syncMsg.users = [];
 			for (let u = 0; u < room.clients.length; u++) {
```

Comment:
`for (let user of room.clients) { ...`

`i` and `u` do not need to be used and are ambiguous.